sdk/core/core-paging/README.md

```diff
@@ -25,14 +25,11 @@ Example of building with the types:
 
 ```typescript
   public listSecrets(
-    options?: ListSecretsOptions
-  ): PagedAsyncIterableIterator<SecretAttributes> {
+    options: ListSecretsOptions = {}
+  ): PagedAsyncIterableIterator<SecretAttributes, SecretAttributes[]> {
```

Comment:
I don't expect to need to specify the array type, but somehow the type `T[]` is not inferred.  Will log an issue for investigation.

Reply:
I think we can change the d.ts to have the generic default to T[] instead of never

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/7568. Could it be a compiler bug?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/7569